command/agent/cache/cachepersist/bolt.go

```diff
@@ -0,0 +1,216 @@
+package cachepersist
+
+import (
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/hashicorp/go-hclog"
+	bolt "go.etcd.io/bbolt"
+)
+
+const (
+	// Keep track of schema version for future migrations
+	storageVersionKey = "version"
+	storageVersion    = "1"
+
+	// CacheFileName - filename for the persistent cache file
+	CacheFileName = "vault-agent-cache.db"
```

Comment:
```suggestion
	DatabaseFileName = "vault-agent-cache.db"
```

Reply:
Hmm, both make sense to me, though I do see this sort of thing referred to as dbpath and dbfile in the raft package, so ok.